code/game/objects/items/weapons/RCD.dm

```diff
@@ -290,7 +290,7 @@ RCD
 		src.spark_system.start()
 	return
 
-/obj/item/weapon/rcd/proc/activate()
+/obj/item/weapon/rcd/activate()

```

Comment:
Maybe change the name of these procs instead of just overriding the assembly stuff?